kwsStyle.cxx

```diff
@@ -284,30 +284,30 @@ int main(int argc, const char **argv)
       return 0;
       }
     file.seekg(0,std::ios::end);
-    unsigned long fileSize = file.tellg();
+    unsigned long fileSize = static_cast<unsigned long> (file.tellg());
```

Comment:
Wouldn't it be better to change the variable type to match whatever `tellg` returns (e.g. `size_type`, `std::streamsize`)? On Windows, this would silently limit file sizes to 4GB.